guides/user/managed-delivery/environment-constraints/index.md

```diff
@@ -287,9 +287,83 @@ If set, overrides the `defaults.constraint.canary.storage-account` property. Not
 
 Apart from the [depends-on](#depends-on) constraint, all constraint types either expect interaction from a user 
 (in the case of [manual-judgement](#manual-judgement)) or can have their automated judgements overruled by a
-user. The following API endpoints are currently available to interact with constraints.
+user.
 
-### Reading status of pending and recent constraints
+### Approving or rejecting Manual Judgement constraints from Slack
+If your operator has configured [Slack app](https://api.slack.com/start/overview) integration with Spinnaker, then a
+very convenient way to approve managed deployments gated by a Manual Judgement constraint is to use interactive Slack
+notifications.
+
+All you need to do is set up Slack notifications in your delivery config, which will enable interactive notifications
```

Comment:
I think you need to add here that the user needs to create a slack channel before, as it does not apply to individual users, no?

Reply:
Thanks for the feedback, Gal. I was under the impression we already had some documentation on how to configure notifications, but I looked again and we do not. I'd prefer to add some more info about that (since it applies beyond constraints) elsewhere and add a link here. How does that sound?

Reply:
I stand corrected: https://www.spinnaker.io/guides/user/managed-delivery/delivery-configs/#environment-notifications

Reply:
Thanks for adding this!